graylog2-server/src/main/java/org/graylog/events/processor/EventDefinitionDto.java

```diff
@@ -54,7 +55,7 @@
 @JsonAutoDetect
 @JsonDeserialize(builder = EventDefinitionDto.Builder.class)
 @WithBeanGetter
-public abstract class EventDefinitionDto implements EventDefinition, ContentPackable<EventDefinitionEntity> {
+public abstract class EventDefinitionDto extends ScopedEntity implements EventDefinition, ContentPackable<EventDefinitionEntity> {
```

Comment:
Should update `toContentPackEntity` to  copy `scope`.

Reply:
Great find! Fixed in https://github.com/Graylog2/graylog2-server/pull/13058/commits/75b7e9fa01a00dc0d175a12754ba9fa83a110ea3